src/DockerContainer.php

```diff
@@ -254,11 +254,11 @@ public function getInspectCommand(string $dockerIdentifier): string
         return implode(' ', $execCommand);
     }
 
-    public function start(): DockerContainerInstance
+    public function start(?float $timeout = 60): DockerContainerInstance
     {
         $command = $this->getStartCommand();
 
-        $process = Process::fromShellCommandline($command);
+        $process = Process::fromShellCommandline($command, null, null, null, $timeout);
```

Comment:
Is there a dedicated method of setting this, so we don't have to pass so many `null`s?